hamocc/mo_read_oafx.F90

```diff
@@ -275,8 +316,9 @@ subroutine get_oafx(kpie,kpje,kplyear,kplmon,omask,oafx)
   real,    intent(out) :: oafx(kpie,kpje)
   integer              :: current_day
```

Comment:
Since there is a section "Local variables" just below, it would be nice to move current_day there.